src/metax_api/models/catalog_record.py

```diff
@@ -44,6 +44,7 @@
 LEGACY_CATALOGS = settings.LEGACY_CATALOGS
 IDA_CATALOG = settings.IDA_DATA_CATALOG_IDENTIFIER
 PAS_CATALOG = settings.PAS_DATA_CATALOG_IDENTIFIER
+ATT_CATALOG = settings.ATT_DATA_CATALOG_IDENTIFIER
```

Comment:
This and the catalog_is_att function is not used which causes the test coverage to fall. should be removed.

Reply:
....Removed